setup.py

```diff
@@ -21,7 +22,8 @@
     "fair>=1.6.1a0,<2.0.0",
     "matplotlib==3.2.2",
     "pyam-iamc",
-    "pymagicc>=2.0.0rc6",
+    # "pymagicc>=2.0.0rc7,<3",
+    "pymagicc @ git+https://github.com/openscm/pymagicc.git@fix-naming",
```

Comment:
Should we roll back this change if openscm/pymagicc#311 is being postponed?

Reply:
Sorry I think we should push openscm/pymagicc#311 through. The one change which isn't in openscm/pymagicc#311  is renaming 'Surface Temperature' to 'Surface Air Tempreature Change'. I keep flip-flopping about whether to include that or not.

Reply:
Hmmm that is a rather large change... I couldn't count the number of times I have typed `Surface Temperature` across our code base

Reply:
Yep, I was thinking we do openscm/pymagicc#311 without Surface Temperature. Then release v2. Then add a deprecation warning that Surface Temperature will change to Surface Air Temperature Change in v3. Then go through that pain next year.

Reply:
That agrees with my comment on openscm/pymagicc#311 so happy to proceed.  I agree we shouldn't make the switch right now.